pkgs/applications/misc/hr/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "hr-${version}";
+  version = "1.2";
+
+  src = fetchFromGitHub {
+    owner = "LuRsT";
+    repo = "hr";
+    rev = version;
+    sha256 = "162vkip2772jl59lschpinimpg4ssiyg7fq0va5cx4d7wldpqmks";
+  };
+
+  phases = [ "unpackPhase" "installPhase" "fixupPhase" ];
+
+  installPhase = ''
+    mkdir -p $out/{bin,share}
+    sed -i "s,PREFIX=.*,PREFIX=$out," Makefile
+    sed -i "s,MANPREFIX=.*,MANPREFIX=$out/share," Makefile
+    make install
```

Comment:
@matthiasbeyer variables in a Makefile can be still overwritten, when in the form `KEY=<value>` via command-line.